include/dsVideoDevice.h

```diff
@@ -388,9 +387,8 @@ dsError_t dsGetCurrentDisplayframerate(intptr_t handle, char *framerate);
  * For source devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
  *
  * @param[in] handle    - The handle returned from the dsGetVideoDevice() function
- * @param[in] framerate - Framerate value to be set frame will be represented in FPS. 
- *                        Please refer ::dsVideoFrameRate_t for  max and min framerate.
- *                        Expects the value as a string(eg:"60").
+ * @param[in] framerate - Framerate value to be set.
```

Comment:
is this frame rate values right?
please check this
https://github.com/rdkcentral/rdk-halif-device_settings/blob/feature/issues72_FrameRate_Mode_desc_update/include/dsAVDTypes.h#L508

Reply:
supported framerates to be updated in description
**ex: 1920x1080x24**